src/charts/line/line.js

```diff
@@ -1785,6 +1786,24 @@ export default function module() {
         return this;
     };
 
+    /**
+     * Gets or Sets the hasMinimumValueScale property of the chart, making yAxix bottom value
+     * to adjust to the minimum dataset value.
+     * By default this is 'false'
+     *
+     * @param  {Boolean} _x Desired minimum value flag
+     * @return { hasMinimumValueScale | module} Current hasMinimumValueScale flag or Chart module
+     * @public
+     */
+    exports.hasMinimumValueScale = function (_x) {
```

Comment:
Could you order this alphabetically? It makes it easier to maintain later.
Thanks!

Reply:
Should I move the initial declaration too? it seems that those variables are not alphabetically ordered.
What is the order criterion of these variables?

Reply:
Nope, those are not ordered for now. It started one way, then changed... don't worry about those.

Reply:
ok, I think it should be good now, let me know if this is correct.